thetis/tracer_eq_2d.py

```diff
@@ -89,10 +89,29 @@ def get_bnd_functions(self, c_in, uv_in, elev_in, bnd_id, bnd_conditions):
         elif 'un' in funcs:
             uv_ext = funcs['un']*self.normal
         else:
-            uv_ext = self.corr_factor * uv_in
-
+            uv_ext = uv_in
         return c_ext, uv_ext, elev_ext
 
+    def wd_bathymetry_displacement(self, eta):
```

Comment:
I have changed this so that now the underlying class for TracerTerm is ShallowWaterTerm. Thus TracerTerm already knows about these functions and I have removed them from the definition of TracerTerm